### PR TITLE
Fix 645: add a "test quality" to constants.QUALITIES

### LINKED ISSUE #645: Add a "test quality" to constants.QUALITIES
This code could pull from constants.QUALITIES. Also, we should add the 480p30 to the constants.QUALITIES dictionary as a new entry called "test_quality" or something.

https://github.com/ManimCommunity/manim/blob/b8644e60fdb9d114cb8ed4ad54bc330335bee636/docs/source/manim_directive.py#L148-L169

cc @ManimCommunity/docs

### PR DESCRIPTION
**Pull request by leotrs**
## List of Changes
- Added a "test_quality" key to the `constants.QUALITIES` dict, to be used by the manim directive

## Further Comments
This ended up being a bit messier than I was expecting, but I still think it's worth it.

Fixes #645 

cc. @behackl

### CODE REVIEW

**Review by behackl — APPROVED** [review]
LGTM, thanks for cleaning up the messy code in the directive; I really don't like the situation where we don't have a single source of truth.

File: docs/source/manim_directive.py [hunk 1]
```diff
@@ -150,29 +152,13 @@ def run(self):
...
-                pixel_width = 2560
-                frame_rate = 60
-            elif quality == "fourk":
-                pixel_height = 2160
-                pixel_width = 3840
-                frame_rate = 60
-
+            quality = f'{self.options["quality"]}_quality'
+        else:
+            quality = "test_quality"
```
Comment (kolibril13) [inline docs/source/manim_directive.py:158]: What is the purpose of naming it "test_quality"?
As this is the quality that is used by default in the web examples, why not name it "web_example_quality"

Comment (leotrs) [reply docs/source/manim_directive.py:158]: Oh. Right. `example_quality` does make more sense!

Comment (kolibril13) [reply docs/source/manim_directive.py:158]: As this is something user-facing, I think `example_quality` could lead to some confusion, as they might not know what this is used for.
Therefore, my preferred way would be to name it `web_example_quality` which is a bit more explicit

**Review by kolibril13 — APPROVED** [review]

File: docs/source/manim_directive.py (hunk 1 above)
Comment (kolibril13) [reply docs/source/manim_directive.py:158]: But I don't have a too strong opinion on this.

**Comment by kolibril13** [comment]
I approved it, but I think test_quality should be still renamed.

File: docs/source/manim_directive.py (hunk 1 above)
Comment (leotrs) [reply docs/source/manim_directive.py]: Done

**Comment by leotrs** [comment]
Renamed and merged.

**Pull request merged**